src/atom_finder/classifier/curly-braces.clj

```diff
@@ -0,0 +1,30 @@
+(in-ns 'atom-finder.classifier)
+(import '(org.eclipse.cdt.core.dom.ast IASTCompoundStatement IASTIfStatement IASTForStatement IASTWhileStatement))
+
+
+(defn curly-braces?
+  "Does this note represent curly brace?"
+  [node]
+  (instance? IASTCompoundStatement node))
+  
+
+(defn curly-braciable-statement?
+  "Does this AST node an statement that should be followed by curly braces?"
+  [node]
+  (let [statement #{;statement that should be followed by compound statements (curly braces)
+                    IASTIfStatement IASTForStatement IASTWhileStatement}]
+      (cond
+        (leaf? node) false
+        (some #(instance? % node) statement) (not-any? true? (map curly-braces? (children node)))
+        :else false)))
+          
+(def curly-braces-atom? curly-braciable-statement?)
```

Comment:
This is actually bad practice, I had done it originally before I had the classifier.clj registry. There's no need to name the top-level function anything special any more. Eventually I'll go back and clean up my aliases as well.